src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs

```diff
@@ -434,14 +439,12 @@ public List<HostComboboxEntry> KnownHosts
 
         private List<string> _selectedHosts = new List<String>();
         /// <summary>
-        /// Current selected hosts as dependencies
+        /// Current selected hosts as dependencies.
+        /// Will be passed for serialization when publishing package.
         /// </summary>
         public List<string> SelectedHosts
         {
-            get
-            {
-                return _selectedHosts;
-            }
+            get { return _selectedHosts; }
```

Comment:
do you mind if we do not use the`_` for fields see:
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/general-naming-conventions

Reply:
Not at all